fractal_server/app/routes/api/v1/task_collection.py

```diff
@@ -143,7 +143,7 @@ async def _insert_tasks(
     """
     Insert tasks into database
     """
-    task_db_list = [Task.from_orm(t) for t in task_list]
+    task_db_list = [Task.model_validate(t) for t in task_list]
```

Comment:
I'm wondering why we sometimes use `Model.from_orm(data)` (or its future replacement) and sometimes we use `Model(**data)`.

If they are equivalent, then let's always use the same one (that is, let's remove `from_orm`).